Integrator/Integrator.Tests/SettingTests/SettingsTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Integrator.Features.Settings.Models;
+using Integrator.Infrastructure;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using NUnit.Framework;
+
+namespace Integrator.Tests.SettingTests
+{
+    public class SettingsTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+    
+        [Test]
+        public void Test_ProperCall()
+        {
+            // Arrange
+            var testObject = new Setting();
+
+            var context = new Mock<IntegratorContext>();
```

Comment:
De her context mocks, er de anderledes end den sqllite mock i har lavet unitOFWork test?
```cs
context = TestHelper.MakeTestDbContext();
```

Reply:
Ja - det er til at lave en test DbContext med in-memory sqlite. Måske den skal navngives anderledes. Måske: `MakeInMemoryDbContext()`.